presto-product-tests/src/main/java/io/prestosql/tests/hive/TestAllDatatypesFromHiveConnector.java

```diff
@@ -398,10 +398,7 @@ public void testSelectAllDatatypesParquetFile()
                         234.567,
                         new BigDecimal("346"),
                         new BigDecimal("345.67800"),
-                        isHiveWithBrokenAvroParquetTimestamps()
```

Comment:
we no longer consider hive parquet behavior as "broken", do we @dain ?
since we added config flag  to replicate hive's behavior

the `isHiveWithBrokenAvroParquetTimestamps` needs be renamed 

- we could tell Avro and Parquet apart, as Parquet works now and Avro still does not (maybe just remove "Parquet" from method name)
- we should have an issue to fix Avro (add a flag like we did for Parquet) and refernce here


Also, @dain the hive's behavior we replicated, is it relative to Hive with https://issues.apache.org/jira/browse/HIVE-21002 or before that?